src/Components/Notification/index.tsx

```diff
@@ -0,0 +1,192 @@
+import { RightArrow } from '@/Assets/RightArrow';
+import { NOTIFICATIONS } from '@/Constants/queryString';
+import { NotificationResponse } from '@/Hooks/notifications/useNotifications';
+import { useReadNotification } from '@/Hooks/notifications/useReadNotification';
+import { NotificationSSEType } from '@/Types/notifications';
+import { getElapsedTime } from '@/utils/date';
+import { moveToDest } from '@/utils/moveToDest';
+import { useQueryClient } from '@tanstack/react-query';
+import { Link } from 'react-router-dom';
+import { MouseEvent } from 'react';
+import styled from 'styled-components';
+import { media } from '@/Styles/theme';
+import { getNotificationTypeText } from '@/utils/getText';
+
+interface NotificationProps<T extends NotificationSSEType> {
+  /** 알림 타입 */
+  type: T['type'];
+
+  /** 알림 id */
+  notificationId: number;
+
+  /** 알림 Title */
+  title: string;
+
+  /** 알림 상세 내용 */
+  content?: string;
+
+  /** 읽은 기록  */
+  read: boolean;
+
+  /** 페이지 이동에 필요한 params */
+  params: T['params'];
+
+  /** 알림 생성 시간 */
+  createdAt: string;
+}
+
+export const Notification = <T extends NotificationSSEType>(props: NotificationProps<T>) => {
+  const { type, notificationId, title, content, read, params, createdAt } = props;
+
+  const { mutate: readMutate } = useReadNotification(notificationId);
```

Comment:
이부분도 수정이 필요할 것 같습니다. 읽음 처리 명세가 바뀐것을 제가 반영했어야 되는데 꼼꼼하지 못했네요 ㅠㅠ 🙏
```suggestion
  const { mutate: readMutate } = useReadNotification([notificationId]);
```

Reply:
이 부분도 반영했습니다! 🙏